integrations/pytorch/recipes/sparse-transfer-learn.md

```diff
@@ -0,0 +1,98 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+---
+# Epoch and Learning-Rate variables
+num_epochs: &num_epochs 10.0
```

Comment:
Should we be including this in a code block?

Reply:
this is a recipe, so the YAML portion is in the front matter (enclosed by `---`)

Reply:
Will leave it as is for now. Will agree the codeblock makes rendered md look way nicer.